apps/console/src/extensions/application-templates/templates/single-page-application/models.ts

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2020, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+ * Copyright (c) 2020-2023, WSO2 LLC. (https://www.wso2.com).
```

Comment:
Do we need to include only this copyright changes since you haven't change anything on this file?

Reply:
Removed the `-2023` part as the file was not edited except the license header change. Fixed with 5bdfc8b